kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -179,7 +179,9 @@ define([
 
             // Set up the link to the web upload app.
             this.$elem.find('.web_upload_div').click(() => {
-                this.initImportApp('web_upload');
+                return Import.setupWebUploadCell().then(() => {
```

Comment:
Wrapped a version of the setup cell distinctly for the web uploader app, so it doesn't need to do anything magical with inputs.